app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -119,6 +119,10 @@ def hosts
         @total = @job_invocation.targeting.hosts.size
         @hosts = @hosts.search_for(params[:search], :order => params[:order]).paginate(:page => params[:page], :per_page => params[:per_page])
         @subtotal = @hosts.respond_to?(:total_entries) ? @hosts.total_entries : @hosts.sizes
```

Comment:
If this needs to be different, can you put this in the `else` branch below to avoid needlessly calculating the subtotal?